optuna/integration/allennlp/_executor.py

```diff
@@ -101,7 +101,11 @@ class AllenNLPExecutor(object):
         serialization_dir:
             A path which model weights and logs are saved.
         metrics:
-            An evaluation metric for the result of ``objective``.
+            An evaluation metric. `Trainer <https://docs.allennlp.org/
+            main/api/training/gradient_descent_trainer/#train>`_ of AllenNLP
+            returns a dictionary consists of metrics after training.
```

Comment:
A grammatical suggestion.
```suggestion
            returns a dictionary containing metrics after training.
```